testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GradleBuildCrossVersionTestConfigurationCacheSmokeTest.groovy

```diff
@@ -17,7 +17,9 @@
 package org.gradle.smoketests
 
 import org.gradle.testkit.runner.TaskOutcome
+import spock.lang.Ignore
 
+@Ignore("Temporarily for Kotlin 2.0.0 upgrade until a wrapper update happens because embedded integ tests are skipped")
```

Comment:
How about using something like "assume(<current date is bigger than let's say next week")" instead so that we make sure we can't forget about this. (I personally might forget...)

Reply:
This was a temporary step, already reverted as part of this PR.